src/renderer/atlas/atlas.vcxproj

```diff
@@ -7,10 +7,12 @@
     <ProjectName>RendererAtlas</ProjectName>
     <TargetName>ConRenderAtlas</TargetName>
     <ConfigurationType>StaticLibrary</ConfigurationType>
+    <IncludePath>$(SolutionDir)\dep\DirectXTK\Inc;$(IncludePath)</IncludePath>
   </PropertyGroup>
   <Import Project="$(SolutionDir)src\common.build.pre.props" />
   <Import Project="$(SolutionDir)src\common.nugetversions.props" />
   <ItemGroup>
+    <ClCompile Include="$(SolutionDir)\dep\DirectXTK\Src\WICTextureLoader.cpp" />

```

Comment:
Alright, my only complaint is this one.

When I first saw the submodule, I was like ... hey okay we're bringing in a big library with a bunch of code to do a specific thing. My first instinct was to say, "Let's use a nuget package if one exists; that'll help us avoid bringing in too much of somebody else's code"

Now that it looks like it's just one file, I'm more inclined to say...

_Let's just copy it into `dep/DirectXTK` and give it a `cgmanifest` just like we did with `dep/jsoncpp`._ If it's really one file, it will be much easier for us to carry it around than a submodule, and if we choose to bring it into Windows we can move it from `dep` to `oss` (the Windows build system doesn't do submodules!)